src/main/java/com/khutircraftubackend/product/ProductController.java

```diff
@@ -18,64 +21,75 @@
 @RequiredArgsConstructor
```

Comment:
similar to category and other api controllers, this is our v1 of "Product Controller".
and similar to other controllers, let's name it in singular: "/v1/product" - this is gonna be our api naming convention for all our controllers.
note, it is not a common / standard approach, but, we have already started it with other controllers, so let's be consistent here as well.
at the same time, table in database usually named in plural :"products" - this is common practice for table names in databases.

Reply:
Regarding the fact that we already use the plural in the names of the controllers, I have an objection, because I used the plural from the beginning of the project. But if you need to change for a joint convention, then ok.

Reply:
ok, then we will have to normalize other controllers.
there are many votes for plural names, examples:
- https://stackoverflow.com/a/32257339
- https://medium.com/@ionut-anghel/8d59b968da36
- https://www.bomberbot.com/rest-api/rest-apis-the-great-debate-over-singular-vs-plural-naming/ 

this will be the kira taosk for it: [SCRUM-79](https://hutircraftfood.atlassian.net/browse/SCRUM-79)

Reply:
I am familiar with some of the posts, but I think that it is better to adhere to your comments, given the experience.